src/UglyToad.PdfPig/Graphics/Operations/PathConstruction/AppendRectangle.cs

```diff
@@ -52,7 +52,7 @@ public void Run(IOperationContext operationContext)
         {
             operationContext.BeginSubpath();
             operationContext.CurrentPath.Rectangle(LowerLeft.X, LowerLeft.Y, Width, Height);
-            operationContext.CurrentPath.ClosePath();
+            //operationContext.CurrentPath.ClosePath();
```

Comment:
If possible delete this commented out code please. I'm just going to check why the tests are failing because I think it might be related.